### ISSUE TITLE
Apple considered privacy-respecting

### ISSUE THREAD

**Issue by samuei** [nikitavoloboev/privacy-respecting#60]
This list seems to implicitly assume that Apple respects privacy, which is at odds with [the FSF's analysis](https://www.gnu.org/proprietary/malware-apple.html#surveillance).

**Comment by nikitavoloboev** [comment]
That is true. I suppose the logic to adding it to the privacy respecting section is that Apple's business model doesn't favor them using personal data for advertising. I am open to PR to clarify this.
